src/com/epam/javatrainings/classdecomposition/pizzeria_palmetto/services/PizzaService.java

```diff
@@ -0,0 +1,37 @@
+package com.epam.javatrainings.classdecomposition.pizzeria_palmetto.services;
+
+import com.epam.javatrainings.classdecomposition.pizzeria_palmetto.model.Ingredients;
+import com.epam.javatrainings.classdecomposition.pizzeria_palmetto.model.Pizza;
+import com.epam.javatrainings.classdecomposition.pizzeria_palmetto.model.PlacingAnOrder;
+
+public class PizzaService implements IngredientsCreator {
+    private final Pizza pizza;
+    public PizzaService(Pizza pizza) {
```

Comment:
the pizzaService shouldn't depend on pizza,
 it can create a pizza, modify(add or remove some ingredient)

Reply:
done